src/foam/u2/wizard/StepWizardletStepsView.js

```diff
@@ -168,15 +167,22 @@ foam.CLASS({
         }))
     },
     function renderSectionLabel(elem, section, index, isCurrent) {
-      let title = section.navTitle;
-      if ( ! title || ! title.trim() ) title = "Part " + index;
-      return elem       
+      let title = section.title;
+      if ( ! title || ! foam.Function.isInstance(title) && ! title.trim() ) title = "Part " + index;
```

Comment:
Could you not call .add() in this 'if' block to avoid creating a slot when not needed?

Reply:
A slot isn't created if title isn't a function.